src/modules/task.js

```diff
@@ -1,5 +1,5 @@
 class Task {
-  constructor(description, completed, index){
+  constructor(description, completed, index) {
```

Comment:
- [ ] When I tried to add a new item, I got this error from the console 👇 

![image](https://user-images.githubusercontent.com/63473584/179379729-3f613730-bfa3-40bf-bcc9-388d65d3cf22.png)

from 👇 

![image](https://user-images.githubusercontent.com/63473584/179379746-72cdc2b1-18cf-47d5-9184-2d2af583cd65.png)

I  see you were trying to access the description attribute at the call of the addTask() function which has not been created yet hence why it is throwing an error and unable to add a new task. Kindly check it out 👍

Reply:
Thank you for the feedback! I'm not sure why I can't re-create the issue. Nevertheless, I'm going to submit another review request after committing the likely fix.